core/admin/mailu/internal/nginx.py

```diff
@@ -93,8 +93,8 @@ def handle_authentication(headers):
             app.logger.warn(f'Received undecodable user/password from nginx: {raw_user_email!r}/{raw_password!r}')
         else:
             try:
-                user = models.User.query.get(user_email)
-                is_valid_user = True
+                user = models.User.query.get(user_email) if '@' in user_email else None
+                is_valid_user = bool(user)
```

Comment:
I'd do it this way: `is_valid_user = user is not None`
But I think this is only a matter of taste.

Reply:
d723326b